viahtml/checkmate/response_schema.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+
+    "title": "Checkmate response object",
+    "$comment": "This schema only enforces the parts we care about and lets the rest drift.",
```

Comment:
There's more to the structure than we enforce here, but there's no point in validating a bunch of things we don't rely on.